docs/docs/admin/monitoring/logging/explorer.md

```diff
@@ -0,0 +1,119 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Grafana Explorer
+
+The Grafana Explorer can be used to view the logs of different components, such
+as the CCM backend or user sessions. The logs are stored persistently, so the
+administrator can also view the logs of already terminated sessions, for
+example to investigate an unexpected session termination.
+
+The Grafana Explorer is available to administrators and can be accessed from
+the Grafana link in the main menu. Select "Explore" to access the log explorer:
+
+![Explore in the main Grafana menu](./explore.png){:style="width:300px"}
+
+## Select the Correct Source
+
+Before you can view logs, you must make sure that the correct source is
+selected for the CCM. For this please select **Loki (Capella Collaboration
+Manager)** in the data source picker.
+
+![Select CCM in Data Source Picker](./data_source_picker.png){:style="width:500px"}
+
+## Labels
+
+The application logs are stored in loki, which uses labels to store and
+retrieve logs efficiently. This means that each log in loki has a set of labels
+assigned to it, where a label is essentially a key-value pair.
+
+The Explorer now consists of two main components, which are described below.
+
+### Label Filters
+
+First, we need to select at least one label and a corresponding value to query
+logs. To do this, simply open the **Select Label** drop-down, select a label,
```

Comment:
```suggestion
To query logs, you need to select at least one label and a corresponding value. To do this, simply open the **Select Label** drop-down, select a label,
```
We usually use "we" to refer to the dev team and "you" to refer to the user. I think "you" is clearer here

Reply:
Changed all occurrences to you or, where possible, used the passive form to avoid you / we at all.